main.js

```diff
@@ -172,7 +172,10 @@ function getICal(urlOrFile, user, pass, sslignore, calName, cb) {
     } else {
         // Find out whether SSL certificate errors shall be ignored
         const options = {
-            uri: urlOrFile
+            uri: urlOrFile,
+	    headers: {
+		'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36'
```

Comment:
Please limit the use of a faked `user-agent:` header to the outlook/office365 domain and do not fake it for all connections.